RFCs/0014-ToJsonString.md

```diff
@@ -0,0 +1,124 @@
+# New Intrinsic Function to Convert Template Block to JSON String
+
+* **Original Author(s):**: @mluk-aws
+* **Tracking Issue**: [Tracking Issue](https://github.com/aws-cloudformation/cfn-language-discussion/issues/14)
+
+# Summary
+
+We will support an intrinsic function called `Fn::ToJsonString` that enables developers to convert a template block into an escaped JSON string, which can be used as input values to string-type properties of CloudFormation resources.
+
+# Examples
+
+Here is an example of a resource containing a string-type property using a JSON formatted string.
+```
+"MySecret": {
+    "Type": "AWS::SecretsManager::Secret",
+    "Properties": {
+        "Name": "MySecret",
+        "SecretString": "{\"password\":\"Pa$$word\",\"secretToken\":\"123\"}"
```

Comment:
Is there a different example we can use that doesn't promote dangerous practices like hardcoding secrets in a template? I like the CloudWatch dashboard example at the bottom. Can we use that instead?